public/app/plugins/panel/graph3/plugins/ExemplarsPlugin.tsx

```diff
@@ -17,19 +9,16 @@ interface ExemplarsPluginProps {
 }
 
 // Type representing exemplars data frame fields
-interface ExemplarsDataFrameViewDTO {
+export interface ExemplarsDataFrameViewDTO {
```

Comment:
Why has this type DataFrame in a name? Does not seem like we turn it into a DataFrame actually, do we? And even if we do I don't think it needs to have DataFrame in a name because of it.

Reply:
It's not DataFrame that's referenced here but [`DataFrameView` ](https://github.com/grafana/grafana/blob/c0762b6ddc60dc2c251cd5c1428a031448c357fe/packages/grafana-data/src/dataframe/DataFrameView.ts#L16) which accepts a generic that represents the shape of the "viewed" DataFrame

Reply:
But you need a DataFrame to create DataFrameView don't you? I mean this is sort of nitpicky but I thought this to be a naming bug that was forgotten or something. Like I would expect this to be something like `ExemplarsData` and whether it is later turned into DataFrame or DataFrameView in some way to be sort of irrelevant.